vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/base_rule.rb

```diff
@@ -0,0 +1,20 @@
+module SecureRooms
+
+  module AccessRules
+
+    class BaseRule
+
+      def self.call(user, card_reader, accounts, selected)
```

Comment:
Hmm, that's an interesting point. I think we only need to run through this list of rules if we're scanning in. Scanning out will probably be a different thing?